src/rgw/rgw_flight.cc

```diff
@@ -688,8 +688,10 @@ arw::Status FlightServer::DoGet(const flt::ServerCallContext &context,
 
   std::unique_ptr<rgw::sal::Bucket> bucket;
 
-  ret = driver->load_bucket(&dp, &(*user), fd.tenant_name, fd.bucket_name,
+  ret = driver->load_bucket(&dp, &(*user),
```

Comment:
sorry, i broke this again in https://github.com/ceph/ceph/pull/54379

`load_bucket()` no longer takes the user argument. you may not need to load `user` here at all?